### PR TITLE
Rename thoth_s2i_* in database schema

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to #2479 

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] No

## This Pull Request implements

Rename `thoth_s2i_*` parts from schema columns to `thoth_*`

### CODE REVIEW

File: thoth/storages/data/alembic/versions/081bee0d9587_create_index_for_thoth_s2i_specific_.py [hunk 1]
```diff
@@ -18,11 +18,11 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_index("thoth_s2i_image_name", "software_environment", ["thoth_s2i_image_version"], unique=False)
+    op.create_index("thoth_image_name", "software_environment", ["thoth_image_version"], unique=False)
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/081bee0d9587_create_index_for_thoth_s2i_specific_.py:21]: this file should not be modified, as these files are automatically created! :)

File: thoth/storages/data/alembic/versions/b957aa9cce3e_introduce_thoth_specific_entries_found_.py [hunk 2]
```diff
@@ -18,13 +18,13 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column("package_extract_run", sa.Column("thoth_s2i_image_name", sa.Text(), nullable=True))
-    op.add_column("package_extract_run", sa.Column("thoth_s2i_image_version", sa.Text(), nullable=True))
+    op.add_column("package_extract_run", sa.Column("thoth_image_name", sa.Text(), nullable=True))
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/b957aa9cce3e_introduce_thoth_specific_entries_found_.py:21]: this file should not be modified, as these files are automatically created! :)

File: thoth/storages/data/alembic/versions/f1824f0c1afa_move_thoth_s2i_information_to_software_.py [hunk 3]
```diff
@@ -18,25 +18,25 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column("external_software_environment", sa.Column("thoth_s2i_image_name", sa.Text(), nullable=True))
-    op.add_column("external_software_environment", sa.Column("thoth_s2i_image_version", sa.Text(), nullable=True))
-    op.drop_column("package_extract_run", "thoth_s2i_image_name")
-    op.drop_column("package_extract_run", "thoth_s2i_image_version")
-    op.add_column("software_environment", sa.Column("thoth_s2i_image_name", sa.Text(), nullable=True))
-    op.add_column("software_environment", sa.Column("thoth_s2i_image_version", sa.Text(), nullable=True))
+    op.add_column("external_software_environment", sa.Column("thoth_image_name", sa.Text(), nullable=True))
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/f1824f0c1afa_move_thoth_s2i_information_to_software_.py:21]: this file should not be modified, as these files are automatically created! :)

File: thoth/storages/data/alembic/versions/081bee0d9587_create_index_for_thoth_s2i_specific_.py (hunk 1 above)
Comment (mayaCostantini) [reply thoth/storages/data/alembic/versions/081bee0d9587_create_index_for_thoth_s2i_specific_.py:21]: Indeed, thanks!

File: thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('has_symbol', 'software_environment_id',
+               existing_type=sa.INTEGER(),
+               nullable=False)
+    op.alter_column('has_symbol', 'versioned_symbol_id',
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:24]: This change and the one below are not related to this even if created automatically.

Comment (pacospace) [reply thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:24]: Actually the automated migration, didn't create a correct almebic migration.

File: thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# revision identifiers, used by Alembic.
+revision = "930b47e27b6c"
+down_revision = "b718c0f8bc53"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("external_software_enviroment", "thoth_s2i_image_name", new_column_name="thoth_image_name")
```
[the remark this answers is not in the document]
Comment (mayaCostantini) [reply thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:21]: Right, thank you!

[the remark this answers is not in the document]
Comment (pacospace) [reply thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:21]: thank you!! 🚀

File: thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Create Date: 2022-01-19 16:56:54.439445+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "930b47e27b6c"
+down_revision = "b718c0f8bc53"
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:14]: same here for the ID

File: thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+# revision identifiers, used by Alembic.
+revision = "930b47e27b6c"
+down_revision = "c427c501f966"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("external_software_enviroment", "thoth_s2i_image_name", new_column_name="thoth_image_name")
```
Comment (pacospace) [inline thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:21]: typo in the name of the columns, here an below: `external_software_environment` and `software_environment`

Comment (mayaCostantini) [reply thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py:21]: Thank you, I'll correct that right away :+1:

**Review by pacospace — APPROVED** [review]
lgtm! tested locally and worked!! 💯

**Pull request merged**